securitycenter/assets/main_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Smoke tests for main.go
+
+package main
+
+import (
+	"os"
+	"testing"
+	"time"
+)
+
+func setup(t *testing.T) string {
+	orgId := os.Getenv("GCLOUD_ORGANIZATION")
```

Comment:
We'll need to add this environment variable to https://github.com/GoogleCloudPlatform/golang-samples/blob/master/testing/kokoro/system_tests.sh. FYI, that will trigger the CI to run all tests, so it will take a bit longer to run after making that change (~30 minutes).

What should the organization be? Is it related to the project this is running in? See https://github.com/GoogleCloudPlatform/golang-samples/blob/master/testing/kokoro/readme.md for a description of the test project setup. Happy to explain more as needed.